c2pa_cbor/README.md

```diff
@@ -0,0 +1,345 @@
+# c2pa_cbor
+
+A fast, lightweight CBOR (Concise Binary Object Representation) encoder/decoder with comprehensive support for tagged types.
+
+## Features
+
+- ✅ Full support for all CBOR major types (0-7)
+- ✅ Tagged types (major type 6) with standard tags:
+  - Date/time strings (tag 0) and epoch timestamps (tag 1)
+  - URIs (tag 32)
+  - Base64url and Base64 encoded data (tags 33, 34)
+  - RFC 8746 typed arrays (tags 64-87) for efficient binary data
+- ✅ Custom tag support via `write_tag()` and `read_tag()` methods
+- ✅ Excellent performance with near-zero overhead
+- ✅ Serde integration for seamless serialization
+- ✅ **Full `serde_transcode` support** - handles `#[serde(flatten)]` and other advanced features
+- ✅ **Backward compatible newtype struct handling** - works with existing CBOR data
+- ✅ **Deterministic encoding** - always produces definite-length CBOR (required for C2PA)
+
+## Installation
+
+Add this to your `Cargo.toml`:
+
+```toml
+[dependencies]
+c2pa_cbor = "0.1"
+serde = { version = "1.0", features = ["derive"] }
+serde_bytes = "0.11"  # For efficient byte array handling
+```
+
+## Quick Start
+
+### Basic Usage
+
+```rust
+use c2pa_cbor::{to_vec, from_slice};
+use serde::{Serialize, Deserialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+struct Person {
```

Comment:
Nitpick: Can't we have a more c2pa-y example, like an asset with its format or mimetype or something like that?

Reply:
yes, now that it is in c2pa. Originally I thought we might want to publish this as an independent crate